Userland/Applications/TextEditor/FileArgument.cpp

```diff
@@ -25,9 +25,9 @@ FileArgument::FileArgument(DeprecatedString file_argument)
     // Match 0 group 2: column number
     if (groups.size() > 2) {
         // Both a line and column number were specified.
-        auto filename = groups.at(0).view.to_deprecated_string();
-        auto initial_line_number = groups.at(1).view.to_deprecated_string().to_int();
-        auto initial_column_number = groups.at(2).view.to_deprecated_string().to_int();
+        auto filename = groups.at(0).view.to_string();
+        auto initial_line_number = groups.at(1).view.to_string().to_number<int>();
+        auto initial_column_number = groups.at(2).view.to_string().to_number<int>();
```

Comment:
Each `to_string()` call will then need `.release_value_but_fixme_should_propagate_errors()` after it.